v_5_2_0/files/config/kubelet-master.yaml.tmpl

```diff
@@ -21,6 +21,7 @@ kubeReserved:
   memory: 1280Mi
   ephemeral-storage: 1024Mi
 kubeReservedCgroup: /kubereserved.slice
+maxPods: 110
```

Comment:
Is it related?

Reply:
its just placeholder now, I will add some logic to set this dynamically for AWS based on instance type